subprojects/core-api/src/main/java/org/gradle/api/invocation/GradleLifecycle.java

```diff
@@ -32,7 +32,10 @@ public interface GradleLifecycle {
     /**
      * Adds an {@link IsolatedAction isolated action} to be called immediately before a project is evaluated.
      *
-     * Any extensions to the {@code Project} model will be available to build scripts.
+     * The isolated action is recreated before being applied to each configured {@code Project}, thus, carrying
+     * shared mutable state across projects is impossible except via {@link org.gradle.api.services.BuildService shared build services}.
```

Comment:
🤔 We are writing here that "shared is impossible except via {@link org.gradle.api.services.BuildService shared build services}." but in fact, this release is not support it, [right](https://github.com/gradle/gradle/pull/28738/files#diff-88b68fb340a3576ded61e7cafd1cf2068c782ac6ef747990cbd887696072669dR123)?

Reply:
Right, I changed the documentation.